packages/expo-screen-capture/android/src/main/java/expo/modules/screencapture/ScreenCaptureModule.kt

```diff
@@ -32,7 +33,6 @@ class ScreenCaptureModule : Module() {
         screenCaptureCallback = Activity.ScreenCaptureCallback {
           sendEvent(eventName)
         }
-        currentActivity.registerScreenCaptureCallback(currentActivity.mainExecutor, screenCaptureCallback!!)
```

Comment:
It could be. AFAIK it's an upstream issue that Kudo has brought up with them. I just made this change for now because this is the only module that stops the app from running on bridgeless.